app/models/setting.rb

```diff
@@ -0,0 +1,14 @@
+##
+# A class to hold site-wide configuration.
+# @todo extract to a configuration file.
+class Setting
+  class << self
+    def uri_endpoint
+      "http://localhost:3000/{?subject}"
+    end
+
+    def root_uri
```

Comment:
With potential confusion as to what "root_uri" refers to (there is a spec test below that uses that terminology for just the application url root) and with the format above bring "uri_<type>", might this be better names "uri_dataset"?